Exercise 12/ourpackages/barcelos.py

```diff
@@ -0,0 +1,107 @@
+import os
+from colorama import Fore 
+
+def kevin():
+    while True:
+        os.system('cls')
+        print(Fore.YELLOW + "\nHello everyone! I am Kevin Joseph V. Barcelos")
+        print(Fore.WHITE + "1. Basic Information")
+        print("2. Goals")
+        print("3. To Do List (Mini task :>)")
+        print("0. To go back")
+        menu_choice = int(input("Enter your choice: "))
+        print(Fore.RESET)
+
+        match(menu_choice):
+            case 1:
+                os.system('cls')
+                print(Fore.BLUE + "\nName: Kevin Joseph V. Barcelos")
+                print("Age: 19")
+                print("Sex: Male")
+                print("Birthday: February 18, 2005")
+                print("Civil Status: Single")
+                input("\nPress Enter to continue.")
+            case 2:
+                os.system('cls')
+                print(Fore.RED + "\n Goals")
+                print("\nLong Term Goals: ")
+                print(Fore.LIGHTYELLOW_EX + "1. Finish College.")
+                print("2. Get a job.'")
+                print("3. Invest in assests.")
+                print("4. Get a house.")
+                print("5. Fall in love.")
+
+                print(Fore.RED + "\nShort Term Goals: ")
+                print(Fore.LIGHTYELLOW_EX + "1. Study Python.")
+                print("2. Pass this semester with good grades.")
+                print("3. Learn Data Analyst Skills.")
+                print("4. Get a part time job.")
+                print("5. Have fun while coding :>")
+                input("\nPress Enter to continue. ")
+            case 3:
+                to_do_list()
+            case 0:
+                break
+            case _: 
+                print("Invalid Choice. Try Again.")
+                input("\nPress Enter to continue.")
+            
```

Comment:
Wait, let me add my own comment in your own module - Marc

Reply:
You have done it :>